### DISCUSSION TITLE
Over 400 errors compiling using MVC 6

### DISCUSSION THREAD

**Discussion by pendraggon87** [Dotnet-Boxed/Templates#60] [Q&A]
When I tried to use the MVC6 template, building came with a LOT of errors.

**Comment by RehanSaeed** [comment]
What are the errors?

Did you follow the instructions for common problems in the ReadMe? Did you:
1. Update to the latest ASP.NET 5 RC1 version.
2. Run `dnvm upgrade`.
3. Close and reopen the the solution.
4. Right click on references and hit restore packages.

**Comment by burtonr** [comment]
Awesome! I had some errors trying to build as well:

In the "project.json" file - had to reorder the dependencies (ReSharper handled this)
In the "Startup.cs", commented out CspConnectSrcAttribute method because it threw an error for the string.Join. "Cannot resolve method 'Join(string, ???)'..." I commented out lines 117, 118, and 120.

Then, restored packages on "Dependencies" and "References" and it worked great!
